internal/attestation/renderer/renderer.go

```diff
@@ -134,6 +152,38 @@ func (ab *AttestationRenderer) Render() (*dsse.Envelope, error) {
 	return &dsseEnvelope, nil
 }
 
+func addPolicyResults(statement *intoto.Statement, policyResults []*v1.Policy) error {
```

Comment:
I'd add a comment explaining why this method needs to leverage structpb and so on